Description: Readline removal
 This patch removes remaining references to readline
Author: Roland Stigge <stigge@antcom.de>

--- magic-7.5.213.orig/commands/Makefile
+++ magic-7.5.213/commands/Makefile
@@ -10,12 +10,7 @@ SRCS      = CmdSubrs.c CmdAB.c CmdCD.c C
 # Force the module to regenerate the symbolic link in the readline
 # directory, as it may be needed by CmdFI.c (in the non-Tcl compile)
 
-module: ${MAGICDIR}/readline/readline lib${MODULE}.o
-
-${MAGICDIR}/readline/readline:
-	@if ( ! test -f ${MAGICDIR}/readline/readline ) ; then \
-	(cd ${MAGICDIR}/readline; ln -s `ls | grep readline` readline) ; \
-	fi
+module: lib${MODULE}.o
 
 include ${MAGICDIR}/defs.mak
 include ${MAGICDIR}/rules.mak
